lib/tasks/biran_tasks.rake

```diff
@@ -7,12 +7,32 @@ namespace :config do
     Rake::Task['config:generate_with_deps'].invoke
   end
 
+  task :generate2 do
+    error_count = 0
+    config.file_tasks.each do |task|
+      Rake::Task["config:#{task}"].invoke
+    rescue Biran::ConfigSyntaxError => e
+      error_count += 1
+      puts e.p_warning
+      next
+    end
+
+    abort 'Errors in creating config files' unless error_count == 0
+    return
+  end
   task :generate_with_deps
 
   config.files_to_generate.each do |file_name, options|
     desc %(Generate the #{file_name}#{options.fetch(:extension, '')} config file)
     task file_name do
       config.create name: file_name, **options
+    rescue ArgumentError => e
+      e_message = "Missing required argument or bad formatting in config file for #{file_name}"
+      e.set_backtrace([])
```

Comment:
This line seems to prevent the backtrace from being printed out. What I wasn't able to do was squash it completely when re-raising some errors of a different type. In those cases, I am happy with the limited output. My argument would be that all you see are code lines from biran code, which don't really point to anything useful anyway. By inserting custom errors into the mix, I believe a more descriptive outcome is now happening that will allow a dev to more quickly narrow down the issue. Especially since we include the name of the block in the errors.